src/boosting/gbdt_model_text.cpp

```diff
@@ -104,7 +104,16 @@ std::string GBDT::DumpModel(int start_iteration, int num_iteration, int feature_
   for (size_t i = 0; i < feature_importances.size(); ++i) {
     size_t feature_importances_int = static_cast<size_t>(feature_importances[i]);
     if (feature_importances_int > 0) {
-      pairs.emplace_back(feature_importances_int, feature_names_[i]);
+      Log::Warning("i = %d, feature_names_.size() = %d", i, feature_names_.size());
```

Comment:
```suggestion
      Log::Debug("i = %d, feature_names_.size() = %d", i, feature_names_.size());
```

Could this please be removed, or made a `DEBUG`--level log? It seems like it was left over from development.

I strongly believe warnings should only be suggestions that could be changed by some user action, not information-only output like this that requires significant knowledge of the project's source code.

If left as-is, users' logs might be filled with stuff like this that they wouldn't understand:

```text
[LightGBM] [Warning] i = 0, feature_names_.size() = 15
[LightGBM] [Warning] i = 1, feature_names_.size() = 15
[LightGBM] [Warning] i = 2, feature_names_.size() = 15
...
```

Reply:
Sorry about that. This should be removed.